test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelpersTest.cs

```diff
@@ -58,6 +60,37 @@ public class TagHelpersTest
 #endif
         }
 
+        [Fact]
+        public async Task CanRenderViewsWithTagHelpersAndUnboundDynamicAttributes_Encoded()
+        {
+            // Arrange
+            var server = TestHelper.CreateServer(_app, SiteName, services =>
+            {
+                _configureServices(services);
+                services.AddTransient<IHtmlEncoder, TestHtmlEncoder>();
```

Comment:
Lets move away from it in a different PR. To use `CommonTestEncoder` I'd have to add a dependency to `Microsoft.AspNet.Testing`. We can do that in one sweeping change since `TestHtmlEncoder` seems to be consistent in the rest of the functional tests.